config/auth.php

```diff
@@ -105,7 +105,21 @@
             'provider' => 'users',
             'table' => 'password_resets',
             'expire' => 60,
+            'throttle' => 60,
```

Comment:
This needs to be removed/unset

Reply:
Imo we should keep it, seems like a sane enough setting

Reply:
Except, it was set before and we had to remove it :)

Reply:
Hmm, git says it wasn't.   We can leave it and see if there are any complaints

Reply:
git doesn't lie!